### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by cliedeman**
Adds github actions as an alternative to appveyor

Includes code in https://github.com/graphql-dotnet/relay/pull/55

Example build: https://github.com/cliedeman/relay/pull/2/checks?check_run_id=671875149

Release is currently disabled.

@sungam3r The publish command might need some tweaks.

### CODE REVIEW

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-2019, ubuntu-latest]
+        dotnetcore:
+          - 2.2.207
```
Comment (cliedeman) [inline .github/workflows/dotnetcore.yml:14]: Not sure if extra versions should be included

File: .github/workflows/dotnetcore.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Install dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+      - name: Test
+        run: dotnet test --no-restore --verbosity normal
+
+  release:
+    runs-on: ubuntu-latest
+    # if: github.ref == 'refs/heads/master'
```
Comment (cliedeman) [inline .github/workflows/dotnetcore.yml:33]: this will need to be un-commented. Not sure how the release strategy should work

**Comment by sungam3r** [comment]
@cliedeman Now I have write access to this project. In a couple of weeks I will have a vacation and I hope to return to #55 . Perhaps this will happen sooner. Keep in mind that I expect to merge #55 first.

**Comment by Shane32** [comment]
Thanks @cliedeman for the submission

GitHub Actions will be in #104 

Fixed readme is in #89 

Other dependency updates were in #80 and #87

**Pull request closed without merging**